changes/2697-sammchardy.md

```diff
@@ -0,0 +1,2 @@
+Update get_field_info_schema to prevent it resetting schema_overrides which stopped **extras on Enum type Fields
+from being saved to the json schema output
```

Comment:
```suggestion
`Enum` fields now properly support extra kwargs in schema generation
```
Or something like that...
No need to be specific in the implementation, more what's the solved issue ;)